pkg/cmd/tknpac/repository/create.go

```diff
@@ -78,18 +81,35 @@ func CreateCommand(run *params.Run, ioStreams *cli.IOStreams) *cobra.Command {
 			gopt.Event.EventType = "[pull_request, push]"
 			gopt.Event.BaseBranch = "main"
 
-			return generate.Generate(gopt)
+			if err := generate.Generate(gopt); err != nil {
+				return err
+			}
+
+			config := &webhook.Webhook{
+				RepositoryURL:  createOpts.gitInfo.URL,
+				PACNamespace:   createOpts.pacNamespace,
+				ProviderAPIURL: githubURLForWebhook,
+			}
+
+			if err := config.Install(ctx, run); err != nil {
+				return err
+			}
+
+			return nil
 		},
 		Annotations: map[string]string{
 			"commandType": "main",
 		},
 	}
 
 	cmd.PersistentFlags().BoolP(noColorFlag, "C", !ioStreams.ColorEnabled(), "disable coloring")
-	cmd.PersistentFlags().StringVar(&createOpts.repository.Name, "name", "", "Repository name")
-	cmd.PersistentFlags().StringVar(&createOpts.event.URL, "url", "", "Repository URL")
+	cmd.PersistentFlags().StringVar(&createOpts.repository.Name, "name", "", "RepositoryURL name")
```

Comment:
is that on purpose?

Reply:
ah my bad !

Reply:
find and replace did something more than expected 😝